spec/models/comment_spec.rb

```diff
@@ -1,5 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validates comments' do
+    subject { Comment.new(text: 'Comment text', author_id: 1, post_id: 2) }
+
+    before { subject.save }
+
+    it 'author_id should be integer' do
+      subject.author_id = 'id'
+      expect(subject).to_not be_valid
+    end
+
+    it 'post_id should be integer' do
+      subject.post_id = 'one'
+      expect(subject).to_not be_valid
+    end
+
+    it 'text should be present' do
+      subject.text = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'author id should be greater or equal to 0 and integer' do
+      subject.author_id = -2
+      expect(subject).to_not be_valid
+    end
+
+    it 'post id should be greater or equal to 0 and integer' do
+      subject.post_id = -1
+      expect(subject).to_not be_valid
+    end
+  end
```

Comment:
- [ ] It will be nice to also write a unit test for this custom method: `comment_updater` used in the `comment.rb` model. Writing unit tests for custom methods used in the model is part of the requirements.